docs/administrators/operational_procedures.rst

```diff
@@ -0,0 +1,216 @@
+Operational Procedures
+----------------------
+
+This document describes the best practices for various operation procedures.
+
+.. note::
+    issue templates are available for all procedures at the bottom of this page
+
+
+TODO: process flowchart
+
+Project Release for Production
+###############################
+
+This process describes the steps to prepare an inmanta project for production release.
+
+Context
+++++++++
+* The project development and testing is complete
+* All modules have been properly released.
+* The project is in a git repo, with a specific branch dedicated to production releases
+* The project is checked out on disk.
+* All modules are checked out on the correct, tagged commit.
+
+Procedure
+++++++++++
+
+1. Verify in `project.yml` that `install_mode` is set to `release`.
+2. (Optional) set the `version` attribute in `project.yml` to the version of this release.
+3. Freeze all modules with
+
+.. code-block:: bash
+
+    inmanta -vv -X project freeze --recursive --operator "=="
+
+3. Verify that all modules are frozen to the correct version
```

Comment:
It is two times 3.

Additionally, what is the effect for the command and how do you see that the versions have been frozen

Reply:
tried to improve